engine/engine/content/builtins/input/default.gamepads

```diff
@@ -341,6 +341,38 @@ driver
     map { input: GAMEPAD_GUIDE type: GAMEPAD_TYPE_BUTTON index: 10 }
 }
 
+driver
```

Comment:
This is the fallback "Standard Gamepad" defined according to the W3C spec. This means that we could theoretically drop almost all of the other gamepads (at least all with exactly the same bindings).